### PR TITLE
fix: i18n of autotest plan list

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
i18n of autotest plan list

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda-org.erda.cloud/erda/dop/projects/387/issues/all?id=272296&issueFilter__urlQuery=eyJ0aXRsZSI6IuOAkOWbvemZheWMluOAkVRlc3QgbWFuYWdlbWVudC9BdXRvIHRlc3QvcGxhbnMiLCJzdGF0ZXMiOls0NDA2LDQ0MDddfQ%3D%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&iterationID=772&type=TASK)

#### Specified Reviewers:

/assign @your-reviewer

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | i18n of autotest plan list             |
| 🇨🇳 中文    |  自动化计划列表的国际化            |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/openapi/component-protocol/scenarios/auto-test-plan-list/components/table/table_test.go [hunk 1]
```diff
@@ -15,17 +15,19 @@
...
 
 	"bou.ke/monkey"
 	"github.com/alecthomas/assert"
-	"golang.org/x/net/context"
 
-	"github.com/erda-project/erda/apistructs"
 	"github.com/erda-project/erda/bundle"
 	protocol "github.com/erda-project/erda/modules/openapi/component-protocol"
+	"github.com/erda-project/erda/pkg/i18n"
+
```
Comment (Effet) [inline modules/openapi/component-protocol/scenarios/auto-test-plan-list/components/table/table_test.go:29]: empty line

**Review by Effet — DISMISSED** [review]

File: modules/openapi/component-protocol/scenarios/auto-test-plan-list/components/table/table_test.go (hunk 1 above)
Comment (chengjoey) [reply modules/openapi/component-protocol/scenarios/auto-test-plan-list/components/table/table_test.go]: adjusted

**Review by sfwn — APPROVED** [review]

**Pull request merged**